domains/media.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ScytheMediaTheDev",
+        "email": "tswartz22@aol.com"
+    },
+    "record": {
+        "MX": ["10"]
```

Comment:
Invalid MX records. No other TXT records present.

Reply:
what MX record should i do

Reply:
@DEV-DIBSTER